tools/generator/src/DTO.swift

```diff
@@ -0,0 +1,3 @@
+struct Project: Equatable, Decodable {
```

Comment:
This file gets a lot bigger, but I wanted to add to it as more tests are added, to keep the churn down if things change while tests/refactoring is happening.